webpage/_data/navigation.yml

```diff
@@ -9,11 +9,17 @@ main:
 site_nav:
   - title: Video Tutorials
     children:
+      - title: "NANOOS Visualization System (NVS)"
+        url: https://www.youtube.com/watch?v=MEVz0jOsqmI
       - title: "IOOS conda environment"
         url: https://github.com/ioos/notebooks_demos/wiki/Installing-Conda-Python-with-the-IOOS-environment
-      - title: "OWSLib"
-        url: https://geopython.github.io/OWSLib/
-      - title: "PyoosS"
-        url: https://github.com/ioos/pyoos
-      - title: "PyCSW"
-        url: http://pycsw.org/
+      - title: "Introduction to Data Explorer"
+        url: https://www.youtube.com/playlist?list=PLBvrtRArn5ffsBARjKsczvfxyYX1wGtFP
+      - title: "Catalog driven reproducible workflows for ocean-science"
```

Comment:
We need a new entry for that because the `url` is the link for the `title`. See https://ioos.github.io/notebooks_demos/

Maybe we should add the link to the paper in the Boston Light Swim notebook.